src/Component/Data/MemberOfAwareExpressionBuilderInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Data;
+
+use Sylius\Component\Grid\Data\ExpressionBuilderInterface as BaseExpressionBuilderInterface;
+
+interface MemberOfAwareExpressionBuilderInterface extends BaseExpressionBuilderInterface
```

Comment:
What is the point of having `memberOf` out of the `ExpressionBuilderInterface`?

Reply:
The DBAL ExpressionBuilder does not support `MemberOf` method, so in this case we have to create separate interface for ORM and PHPCRODM which uses `MemberOf`